.github/workflows/ci.yml

```diff
@@ -24,7 +21,7 @@ jobs:
       uses: actions/setup-java@v2
       with:
         java-version: 11
```

Comment:
in regard to this we are using java 11 here because the detekt extension goes into our pkware conventions plugin which will impact other projects on the DSM side besides core-libraries